app/src/org/commcare/google/services/ads/AdMobManager.java

```diff
@@ -0,0 +1,98 @@
+package org.commcare.google.services.ads;
+
+import android.content.Context;
+import android.view.View;
+import android.widget.FrameLayout;
+
+import com.google.android.gms.ads.AdRequest;
+import com.google.android.gms.ads.AdSize;
+import com.google.android.gms.ads.AdView;
+import com.google.android.gms.ads.MobileAds;
+
+import org.commcare.CommCareApplication;
+import org.commcare.dalvik.BuildConfig;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Controls all actions related to showing ads in consumer apps via AdMob
+ *
+ * @author Aliza Stone
+ */
+public class AdMobManager {
+
+    private static final Map<String, String> adMobIdsMap = initAdMobIdsMap();
+
+    public static final String CLINICAL_SCALES_PACKAGE_ID =
```

Comment:
@ctsims Hm, actually, I think I could set this up such that all of the necessary AdMob information gets included in the same `config.txt` file as all the other consumer apps data that is needed, and then just only do all of the AdMob setup if that information is included. I'll give that a try today unless you tell me otherwise.